src/scss/components/local-footer/_local-footer.scss

```diff
@@ -0,0 +1,14 @@
+@charset "UTF-8";
+
+.su-local-footer__action-links,
+.su-local-footer__primary-links,
+.su-local-footer__secondary-links {
+  a {
+    display: block;
+    margin-bottom: 7px;
+  }
+}
+
+.su-local-footer__social-links {
```

Comment:
should this be an empty class?

Reply:
Nope, it was placeholder that wasn't needed.